app/views/users/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>New User</h1>
```

Comment:
Should we not keep the New User title so that the "admin" user knows that they are creating a new user?